pilot/pkg/security/authz/builder/builder.go

```diff
@@ -136,9 +115,8 @@ func createTCPFilter(config *envoyRbacHttpPb.RBAC) *tcpFilterPb.Filter {
 	// The build function always return the config for HTTP filter, we need to extract the
 	// generated rules and set it in the config for TCP filter.
 	rbacConfig := &envoyRbacTcpPb.RBAC{
-		Rules:       config.Rules,
-		ShadowRules: config.ShadowRules,
```

Comment:
This PR is also trying to clean up all permissive mode policies? What is the current status of permissive mode policy? Is it completely broken already?

Reply:
yes, permissive is not supported in beta policy, it should still work in alpha.